tools/c7n_azure/c7n_azure/storage_utils.py

```diff
@@ -65,8 +65,17 @@ def put_message(self, queue_name, content):
 
     def get_messages(self, queue_name, num_messages=None, visibility_timeout=None):
         queue_service = self._get_service(queue_name)
-        return [m for m in queue_service.receive_messages(number_of_messages=num_messages,
-                                                          visibility_timeout=visibility_timeout)]
+        messages = queue_service.receive_messages(number_of_messages=num_messages,
```

Comment:
In the docs they show this working by setting the page size now, then just getting a single page.

(And add `visibility_timeout` in this also)

```
   # Receive messages by batch
   messages = queue.receive_messages(messages_per_page=5)
   for msg_batch in messages.by_page():
       for msg in msg_batch:
           print(msg.content)
           queue.delete_message(msg)
```

Reply:
yeah, default per page is 32 and I'm sure it was 32 in previous SDK.. So we do exactly this in the code now